### ISSUE TITLE
Inbound calls fail to be answered, "global name 'config' is not defined"

### ISSUE THREAD

**Issue by danry25** [AccelerateNetworks/PagingServer#13]
Since the rewrite after release v0.1, inbound calls are no longer auto-answered, with the error below occurring when an inbound call is received. It appears config needs to be defined in this script.

```
user@debian-PA:~$ sudo service paging status
● paging.service
   Loaded: loaded (/etc/systemd/system/paging.service; enabled)
   Active: active (running) since Mon 2015-08-10 14:20:07 PDT; 47s ago
 Main PID: 17822 (python2)
   CGroup: /system.slice/paging.service
           └─17822 python2 /opt/bin/paging.py

Aug 10 14:20:48 debian-PA paging.py[17822]: 2015-08-10 14:20:48 :: main ERROR :: ERROR (on_incoming_call): global name 'config' is not defined
Aug 10 14:20:48 debian-PA paging.py[17822]: Traceback (most recent call last):
Aug 10 14:20:48 debian-PA paging.py[17822]: File "/opt/bin/paging.py", line 29, in _wrapper
Aug 10 14:20:48 debian-PA paging.py[17822]: try: return func(*args, **kws)
Aug 10 14:20:48 debian-PA paging.py[17822]: File "/opt/bin/paging.py", line 91, in on_incoming_call
Aug 10 14:20:48 debian-PA paging.py[17822]: if config.has_section('pa'):
Aug 10 14:20:48 debian-PA paging.py[17822]: NameError: global name 'config' is not defined
```

**Comment by mk-fg** [comment]
Yeah, hence the:

```
# XXX: both are BROKEN, need to figure out hw audio output first
```

comment around these, didn't get to these at all yet.

**Comment by mk-fg** [comment]
Should be fixed as of d1f93c2.

Note that config changed quite drastically, as should be fairly obvious from [paging.example.conf](https://github.com/AccelerateNetworks/PagingServer/blob/master/paging.example.conf) or [README.rst](https://github.com/AccelerateNetworks/PagingServer/#audio-configuration), but there's still nothing really mandatory but the sip account details.

What IS mandatory now though is having JACK (plus python client module) around, but if it is, probably should start automatically with the script.
No extra configuration should be needed there, at least with one sound card that it'd pick up by default, I guess.

More on up-to-date requirements in the same [README file](https://github.com/AccelerateNetworks/PagingServer/#requirements).
Installation section there is currently MIA.

**Comment by mk-fg** [comment]
Definitely closing this one, the code for mentioned error isn't even there o_O
